amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/common/FiltersEndpoint.java

```diff
@@ -1,792 +1,817 @@
+     * <dt><b>tab</b><dd> - under which tab should be shown the filter tree
+     * <dt><b>items</b><dd> - the name of the object from which the values should be fetched
+     * <dt><b>filtered</b><dd> - if the tree should be build dynamically. 
+     * If it is false, the list of items should be taken as it is. 
+     * </dl></br></br>
+     * 
+     * The items object contains the values used to build the tree.
+     *
+     * </br>
+     * <h3>Sample Output:</h3><pre>
+     * "listDefinitions" : 
+     *  [
+     *    ...
+     *    {
+     *     "id": 1,
+     *     "name": "Donor",
+     *     "displayName": "donor",
+     *     "filterIds": ["donor-type", "donor-group", "donor-agency"],
+     *     "tab": "Funding Organizations",
+     *     "items": "organizations",
+     *     "filtered": false
+     *    },
+     *    {
+     *     "id": 8,
+     *     "name": "Contracting Agency",
+     *     "displayName": "Contracting Agency",
+     *     "filterIds": ["", "contracting-agency-group", "contracting-agency"],
+     *     "tab": "All Agencies",
+     *     "items": "organizations",
+     *     "filtered": false
+     *    }
+     *    ...
+     *  ], 
+     * "items" : {
+     *   "organizations": [
+     *     {
+     *        "id": 4,
+     *        "name": "Bilateral",
+     *        "children": [
+     *            {
+     *               "id": 4,
+     *               "name": "BILATERAL Group",
+     *               "children": [
+     *                  {
+     *                     "id": 20,
+     *                     "name": "Austria",
+     *                     "acronym": "Austria",
+     *                     "listDefinitionIds": [
+     *                        1,
+     *                        2,
+     *                        4,
+     *                        11
+     *                     ]
+     *                  },
+     *                  {
+     *                     "id": 21,
+     *                     "name": "Belgium",
+     *                     "acronym": "Belgium",
+     *                     "listDefinitionIds": [
+     *                        1
+     *                     ]
+     *                  },
+     *                  ...
+     *               ]
+     *            },
+     *            ...
+     *         ]
+     *      },
+     *      ...
+     *   ]
+     *  }
+     * </pre>
+     * 
+     * @return tree definitions (filter types) and the list of organizations
+     */
     @GET
-    @Path("/activityapprovalStatus")
+    @Path("/organizations")
     @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
-    @ApiMethod(ui = true, id = FiltersConstants.APPROVAL_STATUS, columns = ColumnConstants.APPROVAL_STATUS,
-                name = "Approval Status", visibilityCheck = "hasToShowActivityapprovalStatusFilter", tab=EPConstants.TAB_ACTIVITY)
-    @ApiOperation("Return activity status options")
-    public FilterDescriptor getActivityApprovalStatus() {
```

Comment:
FilterDescriptor is the class the is no longer In use

Reply:
Yes, agree. The class was removed.